ros_buildfarm/common.py

```diff
@@ -51,6 +53,8 @@ def get_repositories_and_script_generating_key_files(
             len(build_file.repository_keys)
         repository_urls += build_file.repository_urls
         repository_keys += build_file.repository_keys
+        if hasattr(build_file, "custom_rosdep_urls"):
```

Comment:
Please use single quotes wherever possible. Same multiple times below.

Regarding the `hasattr` I will comment below.